rpcs3/Emu/RSX/VK/VKVertexBuffers.cpp

```diff
@@ -171,7 +171,7 @@ namespace
 				rsx::method_registers.current_draw_clause.primitive,
 				rsx::method_registers.restart_index_enabled(),
 				rsx::method_registers.restart_index(), command.ranges_to_fetch_in_index_buffer,
-				[](auto prim) { return !vk::is_primitive_native(prim); });
+				[](auto prim) { return !vk::is_primitive_native(prim); }, rsx::method_registers.vertex_data_base_index());
```

Comment:
Not a huge fan of predicates dangling in the middle of function lists, to allow expansion onto more than one line without messing up the flow. Move the vertex base index argument before the predicate function. Other than that it should be good to go.